modules/nw-metallb-configure-l2-advertisement-interface.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/about-advertising-ipaddresspool.adoc
+
+:_content-type: PROCEDURE
+[id="nw-metallb-configure-with-L2-advertisement-interface_{context}"]
+= Configuring MetalLB with a L2 advertisement by using interface
+
+By default, the IP address from `IPAddressPool` that has been assigned to the service, is advertised from all the network interfaces. The `interfaces` field in the `L2Advertisement` custom resource definitions is used to restrict those network interfaces which advertise the `IPAddressPool`.
+
+This example shows how to configure MetalLB so that the `IPAddressPool` is advertised only from the network interfaces listed in the `interfaces` field of all nodes.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You are logged in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create an IP address pool.
+
+.. Create a file, such as `ipaddresspool.yaml`, and enter the configuration details.
++
+.Example
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  namespace: metallb-system
+  name: doc-example-l2
+spec:
+  addresses:
+    - 4.4.4.0/24
+  autoAssign: false
+----
+
+.. Apply the configuration for the IP address pool:
++
+[source,terminal]
+----
+$ oc apply -f ipaddresspool.yaml
+----
+
+. Create a L2 advertisement advertising the IP with `interfaces` selector.
```

Comment:
Should this be "an L2", since I assume it's pronounced el-two? I see mixed use across the assembly, so it could probably be cleaned up across at some point, if so.

Reply:
I agree with you! I too am inclined towards 'an'. However, I just kept it 'a' as the previous section used 'a'. I can modify the previous section to 'an' as a part of this item.